src/views/Manager/index.js

```diff
@@ -132,6 +134,7 @@ const updateManager = (payload) => {
             isEditing={isEditing}
             submitHandler={onFormikSubmit}
             cancelHandler={onCancelClick}
+            calendarState={calendarState}
```

Comment:
I don't know if the individual manager page actually needs calendarState, does it? Ultimately I think that whatever gets this page working is a good idea, but it might be confusing for other folks in the future if my hunch is right and this isn't necessary

Reply:
You are correct, Manager does not specifically use it.  However, might be useful in the future, in case something like tenure or hiring dates,etc wanted to be added.

Though, it might be better to pass the same argument that TenantView has.